suzieq/poller/worker/coalescer_launcher.py

```diff
@@ -78,48 +87,45 @@ async def _monitor_coalescer(self):
         """This function calls _start_coalescer() and check if the process
         dies
         """
-        fd = 0
-        # Check to see file lock is possible
-        while not fd:
+
+        starting_attemps = 0
```

Comment:
Typo in name: attemps instead of attempts. Typos can lead to strange, unexpected errors. Please fix it.